yaml/boundaries.yaml

```diff
@@ -95,16 +112,47 @@ filters:
       <<: *output_properties
       kind: {col: kind}
 
+  # OSM disputed boundaries
+  #
+  # NOTE: these come _first_ so that they match in preference to regular
+  # country borders. (this assumes disputed_by is never set on a whole
+  # country boundary -- perhaps true?)
```

Comment:
These 2 are technically entirely disputed_by their neighbours, but instead tagged with claimed_by? How should this be setup / changed (data or processing)?

 **Taiwan:** https://www.openstreetmap.org/relation/449220
**Western Sahara:** https://www.openstreetmap.org/relation/5441968

Reply:
I think I was mistaken when I wrote that comment. I've added a test for this in c2efb38b, and it works, so I removed the comment and replaced it with better tagging recommendations.

For Western Sahara, the tagging looks good to me. I think :crossed_fingers: that it should work the next time we do a build.

For Taiwan, it's missing:

* A `boundary=claim` relation with `claimed_by=CN` and `admin_level=2`. I had a look around, and couldn't find one. It should contain the same ways as [relation 449220](https://www.openstreetmap.org/relation/449220), the country border.
* The ways making up the country border need `disputed=yes` and `disputed_by=CN` on them. I didn't audit all the ways, but a few tested at random (e.g: [this one](https://www.openstreetmap.org/way/53100715)) were missing those tags.

Reply:
Agree I need to add OSM tagging for Taiwan.